### DISCUSSION TITLE
How to get ctrl+N same behaviour as down arrow.

### DISCUSSION THREAD

**Discussion by ChillarAnand** [marlonrichert/zsh-autocomplete#734] [General]
With down arrow I am able to cycle through all the options.

With ctrl+n, on first press, it goes to first selection. When I press again, auto complete stops instead of going to second option.

<img width="642" alt="Screenshot 2024-06-27 at 12 09 04" src="https://github.com/marlonrichert/zsh-autocomplete/assets/4463796/588abb14-4ff6-4098-80df-79d6879dfbab">

<img width="642" alt="Screenshot 2024-06-27 at 12 09 07" src="https://github.com/marlonrichert/zsh-autocomplete/assets/4463796/d8dea0c2-c707-4748-8c36-f067687984f9">

<img width="642" alt="Screenshot 2024-06-27 at 12 09 11" src="https://github.com/marlonrichert/zsh-autocomplete/assets/4463796/71a443da-2395-4195-a712-dd80b5acae26">

**Comment by alexwforsythe** [comment]
Check what `"^N"` is bound to in your `main` and `menuselect` keymaps:
```sh
bindkey -M main -L | grep '"^N"'
bindkey -M menuselect -L | grep '"^N"'
```

It's probably bound to `menu-select` in `main`, which enters the completion menu. Once your cursor is in the menu, the `menuselect` keymap takes over. In `menuselect`, the `menu-complete` widget cycles the cursor to the next item.

These bindings should make <kbd>ctrl</kbd>+<kbd>n</kbd> and <kbd>ctrl</kbd>+<kbd>p</kbd> enter the completion menu and cycle to the next and previous item:
```sh
bindkey '^n' menu-select
bindkey "^p" menu-select
bindkey -M menuselect '^n' menu-complete
bindkey -M menuselect '^p' reverse-menu-complete
```
